README.md

```diff
@@ -27,7 +27,7 @@ Our vision for the future on our [Wiki](https://github.com/nuttyartist/notes/wik
 - Markdown Support. Format text without lifting your hands from the keyboard.
 - Different themes. Switch between Light, Dark, and Sepia.
 - Feed View. Select multiple notes to see them all one after another in the editor.
-- Always runs in the background. Use the hotkey "Windows" + 'N' to summon Notes. "control" + "N" for macOS.
+- Always runs in the background. Use the hotkey <kbd>Win</kbd>+<kbd>Shift</kbd>+<kbd>N</kbd> to summon Notes. <kbd>Ctrl</kbd>+<kbd>N</kbd> for macOS.
```

Comment:
I would suggest to wrap the entire shortcut in `<kbd>` rather than each individual key. Or does that look bad?

Reply:
It would look *kinda* weird like that, in my opinion:

![image](https://user-images.githubusercontent.com/626206/219352428-b9878c30-6ae1-4897-922e-3cc2f39fb3f7.png)

Right now it looks like this:

![image](https://user-images.githubusercontent.com/626206/219352277-3fabea35-c6e8-48c6-8f21-aee9e5568d39.png)

I'm not a designer though, so I could be wrong... lol

Reply:
Yeah, it looks fine like you did it.